packages/formatter-html/scripts/create-get-message.js

```diff
@@ -0,0 +1,43 @@
+const fs = require('fs');
+const path = require('path');
+
+const { format } = require('@hint/utils/dist/src/i18n/format');
+
+const filename = path.join(process.cwd(), 'src', 'assets', 'js', 'scan', 'get-message.js');
+
+const content = `// Autogenerated by scripts/create-get-message.js
```

Comment:
Same here with respect to a comment explaining why this exists. Specifically how this generated function is intended to be passed to `ejs` templates at evaluation time to resolve contained strings.

Also probably worth noting how it depends on `window.localeStrings` being defined via a `messages.js` file compiled by `scripts/compile-locales.js`.

Lastly clarify that this and the generated `message.js` files are only for supporting the online-scanner and will not be used when consuming `formatter-html` via the CLI.

Reply:
I will add the comment but, this files are going to be used also for the formatter-html when it is used via the CLI. Only for a couple of strings, but they are also necessary.

Reply:
Ok, good clarification.